### PR TITLE
SAK-29411 Site browser changes

### PR DESCRIPTION
**Pull request by adrianfish**
The titles now take you into the site and there is an info link which
pops up a dialog with the site info, as in Worksite Setup.

### CODE REVIEW

File: entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java [hunk 1]
```diff
@@ -167,6 +169,20 @@ public EntitySite(String title,
         getUserRoles(); // populate the user roles
     }
 
+    public EntitySite(Site site, boolean includeGroups, boolean justInfo) {
```
Comment (bjones86) [inline entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java]: Seems like you're introducing a new parameter here, justInfo, but it is never used.

Comment (adrianfish) [inline entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java]: I know, but it's just a constructor marker. I could just rename it to markerArgument or something. Thanks for having a look.

Comment (bjones86) [inline entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java]: Right, my point though is if it's not used, why include it? If the only reason is to create a new method signature, there should at least be some javadoc explaining this.

Comment (adrianfish) [inline entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java]: Fair enough. Maybe converting it to a builder pattern would make more sense. Perhaps adding a new method called copyInfoOnly and just using the default constructor?

Comment (bjones86) [inline entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java]: Yeah, that seems like a much better approach to me.

**Comment by adrianfish** [comment]
Updated push. Amended as per conversation with Brian.

**Comment by bjones86** [comment]
Looks good, thanks Adrian.

**Pull request merged**